doc/api/cli.md

```diff
@@ -412,6 +412,16 @@ added: v9.0.0
 Specify the `module` of a custom [experimental ECMAScript Module loader][].
 `module` may be either a path to a file, or an ECMAScript Module name.
 
+### `--insecure-http`
+<!-- YAML
+added: FIXME
```

Comment:
Nit: according to [docs](https://github.com/nodejs/node/blob/master/doc/releases.md#step-3-update-any-replaceme-and-dep00xx-tags-in-the-docs) and [release automation](https://github.com/nodejs/node/blob/master/Makefile#L924-L927), this needs to be
```suggestion
added: REPLACEME
```